atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/AbstractCassandraKeyValueServiceIntegrationTest.java

```diff
@@ -134,8 +118,9 @@ public static Collection<Object[]> data() {
                 AtlasDbConstants.DEFAULT_INITIALIZE_ASYNC);
     });
 
-    public CassandraKeyValueServiceIntegrationTest(String name, UnaryOperator<KeyValueService> keyValueServiceWrapper) {
-        super(CASSANDRA, keyValueServiceWrapper);
+    public AbstractCassandraKeyValueServiceIntegrationTest(
+            UnaryOperator<CassandraKeyValueService> keyValueServiceWrapper) {
+        super(CASSANDRA, kvs -> keyValueServiceWrapper.apply((CassandraKeyValueService) kvs));
```

Comment:
This is the way I found for casting `UnaryOperator<CassandraKeyValueService>` to `UnaryOperator<KeyValueService>`.